packages/appium/docs/en/ecosystem/build-drivers.md

```diff
@@ -581,22 +581,36 @@ call something like:
 driver.executeScript('soundz: playSong', [{song: 'Stairway to Heaven', artist: 'Led Zeppelin'}]);
 ```
 
-Then in your driver code you could simply look for `soundz:` at the beginning of your
-`execute` override, and do the appropriate thing:
+Then in your driver code you could simply define the magic static field `executeOverloadMap` as
```

Comment:
I'm not sure I dig `executeOverload`.  While I appreciate the ultraviolence it evokes, maybe `executeMap` would work just as well?

Reply:
maybe `executeMethodMap` on parity with `newMethodMap`. we could call these `Execute Methods`